frontend/src/app/services/question.service.ts

```diff
@@ -1,92 +1,94 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { Observable, of } from 'rxjs';
+import { map, Observable } from 'rxjs';
+import { environment } from '../../environments/environment.development';
 
 export interface Choice {
   title: string;
+  subtitle?: string;
   score: number;
+  type: string;
 }
 
 export interface Question {
-  id: string;
+  id: number;
   title: string;
+  subtitle?: string;
   choices: Choice[];
 }
 
 @Injectable({
   providedIn: 'root',
 })
 export class QuestionService {
-  private questions: Question[] = [
-    {
-      id: '1',
-      title: 'You find it easy to introduce yourself to other people.',
-      choices: [
-        { title: 'Strongly Agree', score: 5 },
-        { title: 'Agree', score: 4 },
-        { title: 'Neutral', score: 3 },
-        { title: 'Disagree', score: 2 },
-        { title: 'Strongly Disagree', score: 1 },
-      ],
-    },
-    {
-      id: '2',
-      title:
-        'You often get so lost in thoughts that you ignore or forget your surroundings.',
-      choices: [
-        { title: 'Strongly Agree', score: 5 },
-        { title: 'Agree', score: 4 },
-        { title: 'Neutral', score: 3 },
-        { title: 'Disagree', score: 2 },
-        { title: 'Strongly Disagree', score: 1 },
-      ],
-    },
-    {
-      id: '3',
-      title:
-        'You try to respond to your emails as soon as possible and cannot stand a messy inbox.',
-      choices: [
-        { title: 'Strongly Agree', score: 5 },
-        { title: 'Agree', score: 4 },
-        { title: 'Neutral', score: 3 },
-        { title: 'Disagree', score: 2 },
-        { title: 'Strongly Disagree', score: 1 },
-      ],
-    },
-    {
-      id: '4',
-      title:
-        'You find it easy to stay relaxed and focused even when there is some pressure.',
-      choices: [
-        { title: 'Strongly Agree', score: 5 },
-        { title: 'Agree', score: 4 },
-        { title: 'Neutral', score: 3 },
-        { title: 'Disagree', score: 2 },
-        { title: 'Strongly Disagree', score: 1 },
-      ],
-    },
-    {
-      id: '5',
-      title: 'You do not usually initiate conversations.',
-      choices: [
-        { title: 'Strongly Agree', score: 5 },
-        { title: 'Agree', score: 4 },
-        { title: 'Neutral', score: 3 },
-        { title: 'Disagree', score: 2 },
-        { title: 'Strongly Disagree', score: 1 },
-      ],
-    },
-  ];
+  private apiKey = environment.backend.apiKey;
```

Comment:
add this to the http.service.ts so it will be inject on all route

Reply:
Add it already.